src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -96,7 +263,51 @@ public bool TryGetInt32Value(out int value)
         /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
         /// <seealso cref="TokenType" />
         /// </exception>
-        public bool TryGetInt64Value(out long value)
+        public bool TryGetInt64(out long value)
+        {
+            if (TokenType != JsonTokenType.Number)
+            {
+                throw ThrowHelper.GetInvalidOperationException_ExpectedNumber(TokenType);
+            }
+
+            ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
+            return Utf8Parser.TryParse(span, out value, out int bytesConsumed) && span.Length == bytesConsumed;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to a <see cref="uint"/>.
```

Comment:
nit: `...to an <see cref="uint"/>`

Reply:
@WinCPP You are correct. https://www.merriam-webster.com/words-at-play/is-it-a-or-an

Reply:
Agreed, makes sense to me. In that case, lets keep it as `...to a <see cref="uint"/>` and `...to a <see cref="ulong"/>` everywhere, consistently.